tests/multidevice/multidevice.cpp

```diff
@@ -157,6 +175,258 @@ TEST_F(multidevice_ft, stream_two_2_then_1)
     m_device1 = NULL;
 }
 
+#define RETURN_K4A_RESULT_LE(msg1, msg2, v1, v2)                                                                       \
+    if (!(v1 <= v2))                                                                                                   \
+    {                                                                                                                  \
+        printf("%s(%d): ERROR: expected %s <= %s\n %lld vs %lld\n", __FILE__, __LINE__, msg1, msg2, LLD(v1), LLD(v2)); \
+        return K4A_RESULT_FAILED;                                                                                      \
+    }
+
+#define RETURN_K4A_RESULT_EQ(msg1, msg2, v1, v2)                                                                       \
+    if (!(v1 == v2))                                                                                                   \
+    {                                                                                                                  \
+        printf("%s(%d): ERROR: expected %s == %s\n %lld vs %lld\n", __FILE__, __LINE__, msg1, msg2, LLD(v1), LLD(v2)); \
+        return K4A_RESULT_FAILED;                                                                                      \
+    }
+
+#define RETURN_K4A_RESULT_NE(msg1, msg2, v1, v2)                                                                       \
+    if (!(v1 != v2))                                                                                                   \
+    {                                                                                                                  \
+        printf("%s(%d): ERROR: expected %s != %s\n %lld vs %lld\n", __FILE__, __LINE__, msg1, msg2, LLD(v1), LLD(v2)); \
+        return K4A_RESULT_FAILED;                                                                                      \
+    }
+
+#define R_EXPECT_LE(v1, v2) RETURN_K4A_RESULT_LE(#v1, #v2, v1, v2)
+#define R_EXPECT_EQ(v1, v2) RETURN_K4A_RESULT_EQ(#v1, #v2, v1, v2)
+#define R_EXPECT_NE(v1, v2) RETURN_K4A_RESULT_NE(#v1, #v2, v1, v2)
+
+static k4a_result_t open_master_and_subordinate(k4a_device_t *master, k4a_device_t *subordinate)
+{
+    *master = NULL;
+    *subordinate = NULL;
+
+    uint32_t devices_present = k4a_device_get_installed_count();
+    R_EXPECT_LE((int64_t)2, devices_present);
```

Comment:
Another way to do this is

```suggestion
    R_EXPECT_LE(2LL, devices_present);
```

Reply:
Thanks!